src/components/Icon/icons/Twitter.js

```diff
@@ -7,11 +7,14 @@ const Twitter = ({ fill, ...props }) => (
55112 3.17512939,8.48242041 3.33789062,8.77539062 C3.49414141,9.06836084 3.68456919,9.33691284 3.90917969,9.58105469 C4.13379019,9.82519653 4.38606631,10.0416657 4.66601562,10.2304688 C4.49674395,10.2239583 4.33072998,10.2076824 4.16796875,10.1816406 C4.00520752,10.1555988 3.84570391,10.1197919 3.68945312,10.0742188 C3.53320234,10.0351561 3.38183667,9.98470083 3.23535156,9.92285156 C3.08886646,9.86100229 2.94401113,9.79101602 2.80078125,9.71289062 L2.80078125,9.76171875 C2.80078125,10.2630233 2.88541582,10.7366514 3.0546875,11.1826172 C3.22395918,11.628583 3.45507666,12.027342 3.74804688,12.3789062 C4.04101709,12.736981 4.38769331,13.0380847 4.78808594,13.2822266 C5.18847856,13.5263684 5.62304453,13.69401 6.09179688,13.7851562 C6.00716104,13.811198 5.92089888,13.8339843 5.83300781,13.8535156 C5.74511675,13.873047 5.65559941,13.8860677 5.56445312,13.8925781 C5.47330684,13.905599 5.38216191,13.9153646 5.29101562,13.921875 C5.19986934,13.9283854 5.10872441,13.9316406 5.01757812,13.9316406 C4.94596318,13.9316406 4.87760449,13.930013 4.8125,13.9267578 C4.74739551,13.9235026 4.68554717,13.9186198 4.62695312,13.9121094 C4.56184863,13.9055989 4.49674512,13.897461 4.43164062,13.8876953 C4.36653613,13.8779296 4.30468779,13.8697917 4.24609375,13.8632812 C4.37630273,14.2669291 4.56510293,14.6412743 4.8125,14.9863281 C5.05989707,15.3313819 5.34960771,15.6308581 5.68164062,15.8847656 C6.01367354,16.1386731 6.38313599,16.3372389 6.79003906,16.4804688 C7.19694214,16.6236986 7.62499775,16.6985677 8.07421875,16.7050781 C7.72265449,16.9850274 7.34993687,17.2324208 6.95605469,17.4472656 C6.56217251,17.6621104 6.15039277,17.8476555 5.72070312,18.0039062 C5.29101348,18.1536466 4.84668198,18.2675777 4.38769531,18.3457031 C3.92870864,18.4238285 3.45833574,18.4628906 2.9765625,18.4628906 L2.73242188,18.4628906 C2.64778604,18.4628906 2.56640664,18.4596354 2.48828125,18.453125 C2.40364541,18.4466146 2.32063843,18.4401042 2.23925781,18.4335938 C2.1578772,18.4270833 2.07812539,18.4173178 2,18.4042969 C2.45573145,18.6972671 2.93098711,18.9576812 3.42578125,19.1855469 C3.92057539,19.4134126 4.43163799,19.605468 4.95898438,19.7617188 C5.48633076,19.92448 6.02831753,20.0465491 6.58496094,20.1279297 C7.14160435,20.2093103 7.70963252,20.25 8.2890625,20.25 C10.1770928,20.25 11.8469979,19.8968134 13.2988281,19.1904297 C14.7506583,18.4840459 15.9713492,17.5709691 16.9609375,16.4511719 C17.9440153,15.3313746 18.6910782,14.0781319 19.2021484,12.6914062 C19.7132187,11.3046806 19.96875,9.93425156 19.96875,8.58007812 C19.96875,8.53450498 19.9671224,8.48893252 19.9638672,8.44335938 C19.960612,8.39778623 19.9589844,8.35221377 19.9589844,8.30664062 L19.9589844,8.1796875 C19.9589844,8.13411436 19.9557292,8.08854189 19.9492188,8.04296875 C20.1510427,7.89973887 20.3447256,7.74674561 20.5302734,7.58398438 C20.7158212,7.42122314 20.8964835,7.251954 21.0722656,7.07617188 C21.2415373,6.90038975 21.4042961,6.71484473 21.5605469,6.51953125 C21.7167977,6.32421777 21.8632806,6.12565205 22,5.92382812 Z"
```

Comment:
Perfect example of how svgo could help. This is a very long `d` for the Twitter icon. Running this through svgo, you can trim this same icon to 7% of this size: 
![image](https://user-images.githubusercontent.com/442443/67242951-3495b980-f41c-11e9-82bb-f1d974422902.png)

Reply:
ooh cool! I'm looking into this now.